samples/client/petstore/swift5/alamofireLibrary/docs/FakeAPI.md

```diff
@@ -228,7 +228,7 @@ For this test, the body for this request much reference a schema named `File`.
 // The following code samples are still beta. For any issue, please report via http://github.com/OpenAPITools/openapi-generator/issues/new
 import PetstoreClient
 
-let body = FileSchemaTestClass(file: File(sourceURI: "sourceURI_example"), files: [File(sourceURI: "sourceURI_example")]) // FileSchemaTestClass | 
+let body = FileSchemaTestClass(file: {...}, files: [{...}]) // FileSchemaTestClass | 
```

Comment:
Is the `{...}` expected?

Reply:
It's not. I've no clue why it happened. Let me try to do a git bisect over the weekend to identify the commit that results in this change.

Given that this change doesn't cause this particular change (the samples have not been updated) so are you ok for me to merge this to map `decimal` properly in the Swift 5 client generator?

Reply:
Ok, merge it 👍